### PR TITLE
Made mkdocs theme to material

### PR DESCRIPTION
**Pull request by Fir121**
[SITE PREVIEW AT THIS LINK](https://fir121.github.io/coding-bootcamp-2022/)

### PR Overview:
- Set up Theme
- Added some content taken from previous bootcamp repo to the `docs/index.md` markdown file.

### Explanation:
Problem with the previous mkdocs deploy was formatting incompatabilities with markdown and mkdocs in the file `docs/HTML-CSS/README.md`. Mkdocs was processing the html/css code within the code blocks hence altering the format of that page accordingly which was unexpected behaviour and has been fixed since. 

Now we can use either theme, I have gone with the material theme since it seems more uniform, previous theme is also available in the repo and in the yml file, it is simply commented out for now.

As always will need one step after merging this PR to main.

### Step after this PR:

- Once the workflow is completed, if successful, go to REPO > settings > pages, and set deploy from a branch, change the branch to `mkdocs-webpage`.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,21 +1,66 @@
-site_name: ACMBPDC
+site_name: ACM Coding Bootcamp
+docs_dir: './docs'
+remote_branch: mkdocs-webpage
```
Comment (abxhr) [inline mkdocs.yml:3]: Could you change the `remote_branch` to `gh-pages`
```suggestion
remote_branch: gh-pages
```

File: mkdocs.yml [hunk 2]
```diff
@@ -1,21 +1,66 @@
...
-    - '5: CLI & Git': './05-CLI-Git/README.md'
-    - '6: GitHub': './06-GitHub/README.md'
+    - Home: 'index.md'
+    - Sessions:
+      - '1: HTML/CSS': './01-HTML-CSS/README.md'
+      - '2: JavaScript (Basics)': './02-JavaScript-1/README.md'
+      - '3: JavaScript (Advanced)': './03-JavaScript-2/README.md'
+      - '4: APIs': './04-API/README.md'
+      - '5: CLI & Git': './05-CLI-Git/README.md'
+      - '6: GitHub': './06-GitHub/README.md'
```
Comment (abxhr) [inline mkdocs.yml:40]: Comment these pages out for now. We'll be keeping those pages only at the session - week

**Review by abxhr — COMMENTED** [review]
LGTM ✅
Just make these two changes and I'll merge it in

**Comment by Fir121** [comment]
Done ✅

**Review by abxhr — APPROVED** [review]

**Pull request merged**